bokeh/models/widgets/groups.py

```diff
@@ -50,6 +51,10 @@ class Group(AbstractGroup):
     Should items be arrange vertically (``False``) or horizontally
     in-line (``True``).
     """)
+    
+    callback = Instance(Callback, help="""
+    A callback to run in the browser whenever a group is manipulated.
+    """)
```

Comment:
To be precise about I this, I think for now it would be better to add this just to the two subclasses I linked in the issue, not all of the `AbstractGroup` subclasses seem to have the callback machinery. But those two do.